gcp/service-auth/tests/test_get_token.py

```diff
@@ -0,0 +1,39 @@
+import os
+import subprocess
+import time
+from typing import Generator
+
+import pytest
+import logging
+import datetime
+from pathlib import Path
+
+from gcp_service_auth import GcpIdentityToken
+
+dev_target = "https://gcp-genpdf-6lhtms3oua-uc.a.run.app"
```

Comment:
I think this should be a config/env variable

Reply:
Will take it out. My bad.